frontend/src/metabase/parameters/components/ParameterLinkedFilters/ParameterLinkedFilters.unit.spec.tsx

```diff
@@ -44,4 +44,30 @@ describe("ParameterLinkedFilters", () => {
 
     expect(onChangeFilteringParameters).toHaveBeenCalledWith(["p2"]);
   });
+
+  it("should not show linked filter options if the parameter has a custom list source", () => {
+    setup({
+      parameter: createMockUiParameter({
+        id: "p1",
+        name: "P1",
+        values_source_type: "static-list",
```

Comment:
Should we also test for  the`card` source type? You could use `it.each` test to test 2 cases at the same time

Reply:
Done 96c11e3df45e6d9d2d53fe615c70c190d6fe6e6b